src/IotHub/IotHub/IotHub/RemoveAzureRmIotHubEventHubConsumerGroup.cs

```diff
@@ -43,6 +43,8 @@ public class RemoveAzureRmIotHubEventHubConsumerGroup : IotHubBaseCmdlet
         [ValidateNotNullOrEmpty]
         public string Name { get; set; }
 
+        public const string ChangeDesc = "Parameter is being deprecated without being replaced.";
```

Comment:
You need to provide a better customer message.  What should the user do insead?  Is there only one endpoint, is this inferred by the cmdlet, or is there some other mechanism for describing the URI being used?

Reply:
Updated the description